nixos/modules/services/system/nscd.nix

```diff
@@ -143,7 +152,10 @@ in
         RemoveIPC = true;
         PrivateTmp = true;
         # https://github.com/twosigma/nsncd/pull/33/files#r1496927653
```

Comment:
This existing comment is about the `NSNCD_HANDOFF_TIMEOUT` value and should probably be kept with that line.